aiosmtpd/context.py

```diff
@@ -0,0 +1,15 @@
+class SessionContext:
+    peer = None
+    ssl = None
+    host_name = None
+    extended_smtp = False
+
+
+class MessageContext:
```

Comment:
Similarly, I'm wondering if we can find a better name than `MessageContext`, although that does work a little better in this case.  I'm hesitant to call it just `Message` because we usually use that term to describe the actual email message contents (e.g. `email.message.Message`).

Do you like any of these better?

* `Status`
* `Receipt`
* `State`

Then of course the other obvious question is whether these two classes can just be folded into a single class?